server/datastore/mysql/policies.go

```diff
@@ -454,7 +454,7 @@ func (ds *Datastore) CountPolicies(ctx context.Context, teamID *uint, matchQuery
 func (ds *Datastore) CountMergedTeamPolicies(ctx context.Context, teamID uint, matchQuery string) (int, error) {
 	var args []interface{}
 
-	query := `SELECT count(*) FROM policies p WHERE p.team_id = ? OR p.team_id IS NULL`
+	query := `SELECT count(*) FROM policies p WHERE (p.team_id = ? OR p.team_id IS NULL)`
```

Comment:
Is it because of the `LIKE [...]`appended after?

Reply:
exactly, we had the same bug on listMergedTeamPolicies, but didn't update this one.

Reply:
without the parenthesis, the query is mis-processed as `WHERE p.team_id = ? OR (p.team_id IS NULL AND p.name = 'foo')`